spring-integration-amqp/src/test/java/org/springframework/integration/amqp/config/AmqpChannelParserTests.java

```diff
@@ -57,7 +58,7 @@ public void interceptor() {
 				TestUtils.getPropertyValue(channel, "dispatcher"), "maxSubscribers", Integer.class).intValue());
 		channel = context.getBean("pubSub", MessageChannel.class);
 		Object mbf = context.getBean(IntegrationUtils.INTEGRATION_MESSAGE_BUILDER_FACTORY_BEAN_NAME);
-		assertSame(mbf, TestUtils.getPropertyValue(channel, "dispatcher.messageBuilderFactory"));
+		assertNotSame(mbf, TestUtils.getPropertyValue(channel, "dispatcher.messageBuilderFactory"));
```

Comment:
This makes no sense to me; the goal was to use the global MBF for __all__ message builds.